mm/iob/iob_remove_queue.c

```diff
@@ -66,15 +66,20 @@ FAR struct iob_s *iob_remove_queue(FAR struct iob_queue_s *iobq)
           iobq->qh_tail = NULL;
         }
 
+      spin_unlock_irqrestore(NULL, flags);
```

Comment:
Just a suggestion: maybe it is worth of using a dedicated spinlock per memory pool or at least once global mm spinlock?

Reply:
Done.